demos/Run_Stac_Hydra.py

```diff
@@ -0,0 +1,64 @@
+import os 
```

Comment:
What is this file for? Anyway pls use consistent naming case (snake case) on this file if we keep it.